src/components/Forms/FormikInput.tsx

```diff
@@ -4,9 +4,9 @@ import {
   FormControlProps,
   FormLabel,
   Input,
-  Stack,
   useColorModeValue,
```

Comment:
Also, there are lot of unused imports and variables across the app after your changes. Do you mind clearing them out and see if there are any unused imports/variables in each changed file?

Reply:
Ref commit: 168a4cc